frontend/app/src/components/home/PrizeContentBuilder.svelte

```diff
@@ -34,12 +34,15 @@
         cryptoBalance as cryptoBalanceStore,
         cryptoLookup,
     } from "openchat-client";
+    import Checkbox from "../Checkbox.svelte";
+    import Select from "../Select.svelte";
 
     const ONE_HOUR = 1000 * 60 * 60;
     const ONE_DAY = ONE_HOUR * 24;
     const ONE_WEEK = ONE_DAY * 7;
     const client = getContext<OpenChat>("client");
     const dispatch = createEventDispatcher();
+    const streaks = ["3", "7", "14", "30", "100"];
```

Comment:
We _do_ actually have a 365 streak implemented in most places as well.

Reply:
Well nobody has this yet so perhaps we shouldn't yet allow it as a prize gate